src/interface/substrate_query.js

```diff
@@ -428,13 +459,34 @@ module.exports = {
                 } catch (e) {
                     return {'error': e.toString()};
                 }
+	    case 'staking/payee':
+                try {
+		    if (!param2) {
+                        return {'error': 'You did not enter the stash '
+                                + 'address that needs to be queried'};
+                    }
+                    return {'result': await getStakingPayee(api,
+                            param2)};
+                } catch (e) {
+                    return {'error': e.toString()};
+                }
 	    case 'staking/unappliedSlashes':
                 try {
                     return {'result': await getStakingUnappliedSlashes(api,
                             param2)};
                 } catch (e) {
                     return {'error': e.toString()};
                 }
+	    case 'staking/validators':
+                try {
+                    if (!param2) {
+                        return {'error': 'You did not enter the stash '
+                                + 'address that needs to be queried'};
+                    }
+                    return {'result': await getStakingValidators(api,param2)};
```

Comment:
Minor change, can you please add a space between the comma and `param2`. So the line would be 
`return {'result': await getStakingValidators(api, param2)};`